backend/controller/ingress/alias_test.go

```diff
@@ -12,11 +12,11 @@ func TestTransformFromAliasedFields(t *testing.T) {
 	schemaText := `
 		module test {
 			data Inner {
-				waz String alias foo
+				waz String alias http foo
```

Comment:
I'm wondering if we should call the aliasing `json`? It is currently specific to the http ingress, but it is also only to do with JSON fields, not HTTP in general. I've gone back and forth on it, WDYT @wesbillman @worstell ?

Just thinking about it again - it's called "JSON" in each language. Let's call it that here too.

Reply:
yeah I like that too!